org.ekstep.uploadcontent-1.5/editor/uploadapp.js

```diff
@@ -173,6 +178,23 @@ angular.module('org.ekstep.uploadcontent-1.5', []).controller('uploadController'
             $scope.showLoader(false);
             return;
         }
+        if($scope.validateEmptyFile($scope.uploader.getFile(0).name)) {
```

Comment:
create a separate function for this and call it inside it in future we may have other validations as well.